week-04/solutions/hangman.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "assertions"
+
+def hangman(word, letters)
```

Comment:
Is this function is not being called and only used it in test? We should calling the function with inputs.

Reply:
Updated